CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2.5.0
+
+### Features ✨ and improvements 🏁
+
+* Added queryRenderedFeatures support to heatmap layers. ([#10996](https://github.com/mapbox/mapbox-gl-js/pull/10996))
+* Added support for combined usage of line-gradient and line-dasharray paint properties. ([#10894](https://github.com/mapbox/mapbox-gl-js/pull/10894))
+* Added `preclick` event allowing popups to close and open in a new location on one click. ([#10926](https://github.com/mapbox/mapbox-gl-js/pull/10926))
+* Added support for Popup className methods while the popup is not on the map. Direct modifications to the popup container CSS class no longer persist.  ([#10889](https://github.com/mapbox/mapbox-gl-js/pull/10889))
+* Improved collision detection for labels along lines. ([#10918](https://github.com/mapbox/mapbox-gl-js/pull/10918))
+* Added `protected` field to mapbox-api-supported validation. ([#10968](https://github.com/mapbox/mapbox-gl-js/pull/10968))
+
+
+### 🐞 Bug fixes
+
+* Fixed `maxBounds` property not working across the 180th meridian. ([#10903](https://github.com/mapbox/mapbox-gl-js/pull/10903))
+* Fixed `map.getBounds()` returning too-large bounds under some conditions. ([#10909](https://github.com/mapbox/mapbox-gl-js/pull/10909))
```

Comment:
Should we expand a bit _under some conditions_ to be more clear? IIRC it's when the map is highly pitched and low zoom?

Reply:
It's three different conditions:
- When the view is looking off a north/south edge of the map and pitch > 0
- When the horizon is visible
-  When terrain is enabled and there is padding.
I was trying to make it more concise, should I just list all of those out?

Reply:
No that seems a bit much, let's go for what you had then!